moler/cmd/commandchangingprompt.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""
+Generic command class for commands change prompt
+"""
+
+__author__ = 'Marcin Usielski'
+__copyright__ = 'Copyright (C) 2019, Nokia'
+__email__ = 'marcin.usielski@nokia.com'
+
+import abc
+import six
+
+from moler.cmd.commandtextualgeneric import CommandTextualGeneric
+from moler.exceptions import ParsingDone
+
+
+@six.add_metaclass(abc.ABCMeta)
+class CommandChangingPrompt(CommandTextualGeneric):
+    """Base class for textual commands to change prompt."""
+
+    def __init__(self, connection, newline_chars=None, prompt=None, runner=None, expected_prompt=r'^>\s*',
```

Comment:
command will change prompt so in my opinion expected_prompt should not have default value

Reply:
Done